### PR TITLE
feat: add resources sql files

### PR DESCRIPTION
**Pull request by adidoshiws**
### Description
- Added resources sql files

### Checklist

- [x] PR description included
- [ ] `yarn test` passes

### CODE REVIEW

File: resources/v2/02_cabs.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE cabs (
+    id INTEGER NOT NULL PRIMARY KEY,
+    cab_type ENUM('Sedan', 'Hatchback', 'Auto', 'Bike'),
+    cab_number integer NOT NULL UNIQUE,
+    cab_model TEXT NOT NULL,
+    driver_id INTEGER NOT NULL,
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    deleted_at timestamp WITH time zone
+);
```
Comment (tousifws) [inline resources/v2/02_cabs.sql]: This query does not work.   
Create the enum type first then assign it to the table  
https://www.postgresql.org/docs/current/datatype-enum.html

File: resources/v2/01_drivers.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE drivers (
+    id INTEGER NOT NULL PRIMARY KEY,
+    driver_name TEXT NOT NULL,
+    lat DECIMAL(8, 6) NOT NULL,
+    long DECIMAL(9, 6) NOT NULL,
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    deleted_at timestamp WITH time zone
```
Comment (tousifws) [inline resources/v2/01_drivers.sql]: What is this `deleted_at` for?

File: resources/v2/01_drivers.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE drivers (
+    id INTEGER NOT NULL PRIMARY KEY,
```
Comment (tousifws) [inline resources/v2/01_drivers.sql]: Make it an identity column https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-identity-column/

File: resources/v2/03_bookings.sql [hunk 4]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE bookings (
+    id INTEGER NOT NULL PRIMARY KEY,
+    pickup TEXT NOT NULL,
+    destination TEXT NOT NULL,
```
Comment (tousifws) [inline resources/v2/03_bookings.sql]: It should hold location coordinates. Use Postgis extension and set it geometry type  http://postgis.net/workshops/postgis-intro/geometries.html

File: resources/v2/03_bookings.sql [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    user_id INTEGER NOT Null,
+    cab_id INTEGER NOT Null,
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    deleted_at timestamp WITH time zone,
```
Comment (tousifws) [inline resources/v2/03_bookings.sql]: Make it `canceled_at` instead of `deleted_at`

File: resources/v2/02_cabs.sql (hunk 1 above)
Comment (adidoshiws) [reply resources/v2/02_cabs.sql]: changed

File: resources/v2/01_drivers.sql (hunk 2 above)
Comment (adidoshiws) [reply resources/v2/01_drivers.sql]: updated to `canceled_at`

File: resources/v2/01_drivers.sql (hunk 3 above)
Comment (adidoshiws) [reply resources/v2/01_drivers.sql]: done

File: resources/v2/03_bookings.sql (hunk 4 above)
Comment (adidoshiws) [reply resources/v2/03_bookings.sql]: Yes, created the postgis extension & used the geometry type.

File: resources/v2/03_bookings.sql (hunk 5 above)
Comment (adidoshiws) [reply resources/v2/03_bookings.sql]: done

File: resources/v2/01_drivers.sql [hunk 6]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE drivers (
+    id INT NOT NULL PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    driver_name TEXT NOT NULL,
+    location geometry(Point, 4326),
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    canceled_at timestamp WITH time zone
```
Comment (tousifws) [inline resources/v2/01_drivers.sql]: We don't need deleted\_at/canceled\_at for drivers & cabs

File: resources/v2/01_drivers.sql [hunk 7]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE drivers (
+    id INT NOT NULL PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    driver_name TEXT NOT NULL,
```
Comment (tousifws) [inline resources/v2/01_drivers.sql]: just name

File: resources/v2/02_cabs.sql [hunk 8]
```diff
@@ -0,0 +1,15 @@
+CREATE TYPE kind AS ENUM ('Sedan', 'Hatchback', 'Auto', 'Bike');
```
Comment (tousifws) [inline resources/v2/02_cabs.sql]: let's name it cab\_kind

File: migrations/20220512130201-create-cab-tables.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+const { migrate } = require('../server/utils/migrateUtils');
+
+module.exports = {
+  up: queryInterface => migrate(__filename, queryInterface),
+  down: function(queryInterface) {
+    return Promise.resolve();
```
Comment (tousifws) [inline migrations/20220512130201-create-cab-tables.js]: Should have commands to drop tables

File: resources/v2/01_drivers.sql (hunk 6 above)
Comment (adidoshiws) [reply resources/v2/01_drivers.sql]: Yes, removed.

File: resources/v2/01_drivers.sql (hunk 7 above)
Comment (adidoshiws) [reply resources/v2/01_drivers.sql]: Done!

File: resources/v2/02_cabs.sql (hunk 8 above)
Comment (adidoshiws) [reply resources/v2/02_cabs.sql]: yeah, that would be better

File: migrations/20220512130201-create-cab-tables.js (hunk 9 above)
Comment (adidoshiws) [reply migrations/20220512130201-create-cab-tables.js]: right, added the commands

File: migrations/20220512130201-create-cab-tables.js [hunk 10]
```diff
@@ -0,0 +1,11 @@
+const { migrate } = require('../server/utils/migrateUtils');
+
+module.exports = {
+  up: queryInterface => migrate(__filename, queryInterface),
+  down: async function(queryInterface) {
+    await queryInterface.dropTable('drivers');
+    await queryInterface.dropTable('cabs');
+    await queryInterface.dropTable('bookings');
+    return Promise.resolve();
```
Comment (tousifws) [inline migrations/20220512130201-create-cab-tables.js:9]: Like this 
```js
return Promise.all[
  queryInterface.dropTable('drivers'),
  queryInterface.dropTable('cabs'),
  queryInterface.dropTable('bookings')
]

```
https://sequelize.org/docs/v6/other-topics/migrations/

You can also throw error like this
```js
down: () => Promise.reject(new Error('error'))
```

**Review by tousifws — APPROVED** [review]
Resolve the comment

File: migrations/20220512130201-create-cab-tables.js (hunk 9 above)
Comment (adidoshiws) [reply migrations/20220512130201-create-cab-tables.js]: yes done!

**Pull request merged**